packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanel.tsx

```diff
@@ -0,0 +1,175 @@
+
+interface State {
+    isValid?: boolean;
+    name?: string;
+    description?: string;
+    categoryId?: number;
+    label?: string;
+    dataRowSetting?: number;
+}
+
+class DatasetPropertiesPanelImpl extends React.PureComponent<Props & InjectedDomainPropertiesPanelCollapseProps, State> {
+
+    constructor(props: Props & InjectedDomainPropertiesPanelCollapseProps) {
+        super(props);
+
+        this.state = {
+            isValid: true,
+            name: this.props.model.name,
+            description: this.props.model.description,
+            categoryId: this.props.model.categoryId,
+            label: this.props.model.label,
+            dataRowSetting: this.getDataRowSetting(this.props.model)
+        };
+    }
+
+    getDataRowSetting(model: DatasetModel) : number {
+        let dataRowSetting = -1;
+
+        // participant id
+        if (model.keyProperty === undefined || model.keyProperty === null) {
+            dataRowSetting = 0;
+        }
+
+        // participant id and timepoint
+        if (model.keyProperty !== null && (model.keyManagementType === undefined || model.keyManagementType === null)) {
+            dataRowSetting = 1;
+        }
+
+        // participant id, timepoint and additional key field
+        if (model.keyProperty !== null && model.keyManagementType !== null) {
+            dataRowSetting = 2;
+        }
+
+        return dataRowSetting;
+    }
+
+    onCheckBoxChange = (name, checked): void => {
+        // TODO: manage state change of server manage field checkbox in next story
+    };
+
+    onInputChange = e => {
+        const id = e.target.id;
+        let value = e.target.value;
+
+        // special case for empty string, set as null instead
+        if (Utils.isString(value) && value.length === 0) {
+            value = null;
+        }
+
+        this.setState(() => ({ [id]: value }));
+    };
+
+    onCategoryChange = (category) => {
+        let categoryID = category ? category : undefined
+        this.setState(() => ({categoryId: categoryID}))
+    };
+
+    onRadioChange = e => {
+        const name = e.currentTarget.name;
+        const value = e.target.value;
+
+        this.setState(() => ({ [name]: value }));
+    };
+
+    render() {
+        const {
+            model,
+            newDataset,
+            showDataspace,
+        } = this.props;
+
+        const {
+            isValid,
+            name,
+            description,
+            categoryId,
+            label,
+            dataRowSetting
+        } = this.state;
+
+        return (
+            <BasePropertiesPanel
+                {...this.props}
+                headerId={'dataset-header-id'}
+                title={'Dataset Properties'}
+                titlePrefix={model.name}
+                isValid={isValid}
+                updateValidStatus={() => {}} // TODO: in next story
+            >
+                <Row className={'margin-bottom'}>
+                    <Col md={11}/>
```

Comment:
See ListPropertiesPanel and DataClassPropertiesPanel, I think we should add the usage of <HelpTopicURL> here for datasets to add a help link for the "Learn more about designer datasets"